Add documentation about the NMDC Field Notes app (#125)
microbiomedata/docs: content/home/src/howto_guides/submit2nmdc.md

Reviewer: Long term, I want to rename this file to something like `submit_to_nmdc.md` or `submit-to-nmdc.md` so that the name is, in my opinion, more semantically correct. Although `x_to_y` and `x2y` are homophones, my gut tells me that, if we use the name with "to" in it, Google's Search indexing algorithm will treat the page more similarly to how I assume we want it to.

Author: happy to change it now!

Author: https://github.com/microbiomedata/docs/pull/125/commits/2ffce45f6c23f151e451c531fc67d87d83f05f81

Fixed at this commit

Author: changed back! will do this as a separate task to links don't break

## Introduction

> The NMDC has developed 2 products for improving metadata capture. The NMDC Submission Portal (https://data.microbiomedata.org/submission/home) (released April 2022) and NMDC Field notes (currently in beta testing) are designed to lower barriers to capture metadata and adhere to community standards, addressing the critical gap of collecting necessary metadata describing a study and its biosamples. 

Reviewer 2: "2 products" -> "two products" ?  (looks like we currently spell out these numbers)

> The NMDC has developed two products for improving metadata capture. The NMDC Submission Portal (https://data.microbiomedata.org/submission/home) (released April 2022) and NMDC Field notes (currently in beta testing) are designed to lower barriers to capture metadata and adhere to community standards, addressing the critical gap of collecting necessary metadata describing a study and its biosamples. 

Reviewer: Here, I made "NMDC Submission Portal" and "NMDC Field Notes" into hyperlinks. I also shortened the second sentence (lengthening the first sentence as a result). Finally, I used Google Gemini (which is like ChatGPT) to massage the shortened second sentence into something that I find easier to parse.

```suggestion
The NMDC has developed two products for improving metadata capture: the [NMDC Submission Portal](https://data.microbiomedata.org/submission/home) (released April, 2022) and the [NMDC Field Notes](https://microbiomedata.org/field-notes/) app (currently in beta testing). Each product was designed to lower barriers to capturing metadata and adhering to community standards, thereby addressing the critical gap of collecting necessary metadata describing a study and its biosamples.
```

### NMDC Submission Portal

> The Submission Portal is designed using a flexible framework leveraging a new modeling approach called the Linked Data Modeling Language (LinkML) and the template-driven spreadsheet tool DataHarmonizer. It supports several different community standards, such as the Minimal Information about any (x) Sequence (MIxS) standard from the Genomic Standards Consortium (GSC), the PROV standard for provenance metadata, the Proteomics Standards Initiative (PSI) standards for metaproteomics, and the Metabolomics Standards Initiative (MSI) standards for metabolomics. The Submission Portal is an intuitive interface that allows researchers to provide information about their study, the metadata about the study’s biosamples, multi-omics data associated with the study, and whether Department of Energy (DOE) user facility proposals are associated with the data. Updates and new features are continually being implemented as user research provides new insights to improve usability, and as standards are updated and improved. 

Reviewer: I made the following changes here:
- Added a comma before "DataHarmonizer"
- Made "DataHarmonizer" be a link to its GitHub repo
- Added "NMDC" before "Submission Portal" (I see it referenced that way elsewhere in this document)
- Italicized the names of the community standards (to distinguish them from the words around them)

```suggestion
The NMDC Submission Portal is designed using a flexible framework leveraging a new modeling approach called the Linked Data Modeling Language (LinkML) and the template-driven spreadsheet tool, [DataHarmonizer](https://github.com/cidgoh/DataHarmonizer). It supports several different community standards, such as the _Minimal Information about any (x) Sequence_ (MIxS) standard from the Genomic Standards Consortium (GSC), the _PROV_ standard for provenance metadata, the _Proteomics Standards Initiative_ (PSI) standards for metaproteomics, and the _Metabolomics Standards Initiative_ (MSI) standards for metabolomics. The NMDC Submission Portal is an intuitive interface that allows researchers to provide information about their study, the metadata about the study’s biosamples, multi-omics data associated with the study, and whether Department of Energy (DOE) user facility proposals are associated with the data. Updates and new features are continually being implemented as user research provides new insights to improve usability, and as standards are updated and improved. 
```

### NMDC Field notes

> The Field Notes App is designed to bring standard metadata capture into the field. This app is built following the same schema format as the Submission portal and also supports the community standards. When using the Field Notes App, metadata can be captured for your samples while in the field using in app tools and formatting. The Field Notes app can be used offline and provides a direct connection to the NMDC Submission Portal so your metadata is at the ready upon field work completion.

Reviewer: I made these changes here:
- Change "Field Notes App" to "NMDC Field Notes app" (it's an app named "NMDC Field Notes")
- Change "supports the community standards" to "supports community standards" (no "the")
- Change "using in app tools" to "using in-app tools"
- Rephrase "at the ready" to "already there" [in the NMDC Submission Portal]

```suggestion
The NMDC Field Notes app is designed to bring standard metadata capture into the field. This app is built following the same schema format as the Submission portal and also supports community standards. When using the NMDC Field Notes app, metadata can be captured for your samples while in the field using in-app tools and formatting. The NMDC Field Notes app can be used offline and provides a direct connection to the NMDC Submission Portal so your metadata is already there upon field work completion.
```

### Support for DOE User Facilities
The DOE User Facilities JGI and EMSL are key partners for the NMDC because they support the environmental research community. The NMDC team collaborates closely with the JGI and EMSL to support integration of multi-omics data generated across these Facilities, and particularly as part of the Facilities Integrating Collaborations for User Science (FICUS) Program. The Submission Portal has been designed upfront to be compliant with both JGI and EMSL sample submission requirements, ensuring study and biosample information is consistently collected to support interoperability and data reuse. To demonstrate feasibility, several FICUS user projects have been submitted using the Submission Portal with feedback that has informed improvements and new features. Through the Submission Portal, study and biosample metadata is validated against the NMDC schema, with ‘realtime’ checks on data integrity (accuracy, completeness, and consistency). Further developments will also support data embargo information in accordance with the JGI and EMSL Data Policies.

> ## Product Access

Reviewer: Hi @mslarae13, this is as far as I got in my review today. I'll continue next week.

TODO: Update this image
[![](../_static/images/howto_guides/portal_guide/portal_functionality.png)](../_static/images/howto_guides/portal_guide/portal_functionality.png)

> All NMDC products are accessible from the ‘Products’ tab on https://microbiomedata.org/.

Reviewer: Since the quote is not [within another quote](https://wmich.edu/writing/punctuation/quotationmarks), I recommend using double quotation marks.

```suggestion
All NMDC products are accessible from the "Products" tab on https://microbiomedata.org/.
```

TODO: UPDATE THIS IMAGE
[![](../_static/images/howto_guides/portal_guide/ORCiD.png)](../_static/images/howto_guides/portal_guide/ORCiD.png)

NMDC Products require ORCiD authentication for full access and functionality.

## Submission Portal Functionality

https://data.microbiomedata.org/submission/home 

TODO: ADD SUBMISSION PORTAL HOMEPAGE IMAGE

[![](../_static/images/howto_guides/portal_guide/ORCiD.png)](../_static/images/howto_guides/portal_guide/ORCiD.png)

The Submission Portal requires ORCiD authentication to access. If you have already signed in via ORCID, you will not see this screen in the NMDC Submission Portal.

[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)

> Once signed in with an ORCiD, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to NMDC. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point.   

Reviewer: ```suggestion
Once signed in with an ORCiD, you will see an option to "Create New Submission" followed by guidance about the submission process. Below that guidance will be a table showing all submissions you have access to. You can click on any of those submissions to resume working on it.
```

Author: @eecavanna I think you're confused here...
When I said "Details about each section are outlined below." I meant Study, Context, Multiomics data, etc.. the sections of a submission.
Not the table that lists submissions available. I've proposed a different change. Thoughts?

Once signed in with an ORCiD, you will see an option to "Create New Submission" followed by sections to provide information required for submission to NMDC. Details about each section are outlined below. Submitters can return to existing submissions saved under the ORCiD account to resume their work at any point.

Reviewer: Sorry, I neglected to look at the screenshots while reviewing the diff. Thanks for following up with me about this. (We discussed this during today's doc squad meeting.)

> Once signed in with an ORCiD, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to NMDC. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point.   

Author: ```suggestion
Once signed in with an ORCiD, you will see an option to ‘Create New Submission’ with subsequent guidance to provide information required for submission to NMDC. Details about each section are outlined below. Submitters can return to existing submissions saved under the ORCiD account to resume their work at any point.   
```

The real-time validation tool allows submitters to check their filled-in metadata and overall progress as they submit values to ensure the submission will be valid and adhere to the NMDC schema. 

##### Find and Replace

TODO: ADD TEST AND IMAGE

##### Color Legend

[![](../_static/images/howto_guides/portal_guide/color_legend.png)](../_static/images/howto_guides/portal_guide/color_legend.png)

All fields in the metadata template innately fall into one of three color categories: Grey, Yellow, or Purple. Grey, or no highlighted color, indicates optional fields. Required columns are denoted with yellow. These yellow columns must be correctly completed or the submission will not validate. Recommended fields, highlighted purple, are required where applicable. If any of the purple columns provide information relevant to the study, they should be completed. For example, if samples are from a moisture manipulation study, the column ‘watering regimen’ should be filled in to provide context and information about the samples. 

> After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The ‘Column Information’ tool, described above, provides expected structure patterns and examples of valid metadata in order to help remedy errors and invalid inputs.

Reviewer: ```suggestion
After clicking the "Validate" button, cells will become color coded to draw your attention to any invalid entries. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The "Column Information" tool, described above, provides expected structure patterns and examples of valid metadata, which you can refer to when addressing validation errors.
```

> After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The ‘Column Information’ tool, described above, provides expected structure patterns and examples of valid metadata in order to help remedy errors and invalid inputs.

Author: ```suggestion
After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The "Column Information" tool, described above, provides expected structure patterns and examples of valid metadata, which you can refer to when addressing validation errors.
```

## Field Notes Functionality

https://microbiomedata.org/field-notes/

> The NMDC Field Notes App is for mobile device use and is available for beta testing. See (TODO) to sign up!

Reviewer: ```suggestion
The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices. It's currently available for beta testing. See (TODO) to sign up!
```

This app is designed to streamline real-time registration and collection of standardized metadata and field measurements. 

> Upon first use, each section of the app will automatically launch a quick walk-through. You can press the → button to continue the tour or press the X button to end it. To re-initiate the walk through at any time, visit the Settings page and select ‘App Walk-Through’

Reviewer: The button for resettings tours is labeled "Reset tours".

```suggestion
Upon first use, each section of the app will automatically launch a quick tour. You can press the "→" button to continue the tour or press the "X" button to end it. To re-initialize the tours at any time, visit the Settings page and tap "Reset tours".
```

> Available in the app is general fieldwork guidance and recommendations for collecting proper metadata for your sample type(s) to help you plan your fieldwork and metadata collection. We recommend viewing this information before you go out into the field.

Reviewer: When I read the first sentence, I felt overwhelmed. I've rephrased it here.

```suggestion
The "Guide" section of the app contains recommendations about fieldwork in general, and about collecting metadata for specific sample type(s). We recommend viewing it before going out into the field.
```

TODO: Add screen shot images

> The Field Notes App requires ORCiD authentication to access. 

Reviewer: ```suggestion
The NMDC Field Notes app requires ORCiD authentication to access it. 
```

After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The ‘Column Information’ tool, described above, provides expected structure patterns and examples of valid metadata in order to help remedy errors and invalid inputs.

## Field Notes Functionality

https://microbiomedata.org/field-notes/

The NMDC Field Notes App is for mobile device use and is available for beta testing. See (TODO) to sign up!

This app is designed to streamline real-time registration and collection of standardized metadata and field measurements. 

> Each section of the app will automatically launch a quick walk-through. You can press the → button to continue the tour or press the X button to end it. To re-initiate the walk through at any time, visit the Settings page and select ‘App Walk-Through’

Reviewer 2: "Each section" -> "Upon first use, each section"

> Each section of the app will automatically launch a quick walk-through. You can press the → button to continue the tour or press the X button to end it. To re-initiate the walk through at any time, visit the Settings page and select ‘App Walk-Through’

Author: ```suggestion
Upon first use, each section of the app will automatically launch a quick walk-through. You can press the → button to continue the tour or press the X button to end it. To re-initiate the walk through at any time, visit the Settings page and select ‘App Walk-Through’
```

Available in the app is general fieldwork guidance and recommendations for collecting proper metadata for your sample type(s) to help you plan your fieldwork and metadata collection. We recommend viewing this information before you go out into the field.

TODO: Add screen shot images

The Field Notes App requires ORCiD authentication to access. 

> Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 

Reviewer 2: Maybe "Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi" could be "Data entered via Field Notes are automatically synced and available in the Submission Portal when connected to wifi or cellular network".
To be check by @pkalita-lbl , but I believe this sync is active as soon as there is an internet connection, no matter if wifi or else ?

Author: @simroux data, or metadata? i worry about confusing here... and study information which isn't really data or metadata also syncs

Reviewer 2: Yeah, I used Data by default but you're right it's not necessarily a good idea...
Maybe "Information entered is" or "Submissions via Field Notes are" (which was the original wording and worked I think) ?

TODO: Update all these to field notes!

(Step 1 of 3) Welcome to the NMDC Field Notes mobile app! 

[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)

> Once signed in with an ORCiD, you will have the ability to access existing studys or create a study using the 'New’ button. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point. 

Reviewer 2: "studys" - "studies"

Author: ```suggestion
Once signed in with an ORCiD, you will have the ability to access existing studies or create a study using the 'New’ button. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point. 
```

### New Study

[![](../_static/images/howto_guides/portal_guide/study_info.png)](../_static/images/howto_guides/portal_guide/study_info.png)

The Study Information page requires a valid ‘Study Name’ and a valid email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [header](#submission-portal-functionality) [header](#study) for more details about Study information.

> Also in the New Study you will select your "Environment Template". This will correspond to the GSC Environmental Extension. More information is available above in sections [header](#environmental-package). Multiple environments can be selects and submitted in one submission.

Reviewer 2: "the GSC Environmental Extension." -> "the GSC Environmental Extension that best fits your samples." ?
"can be selects" -> "can be selected"

Author: ```suggestion
Also in the New Study you will select the "Environment Template" that best fits your samples. The templates available correspond to the GSC Environmental Extensions. More information is available above in sections [header](#environmental-package). Multiple environments can be selected and submitted in one submission.
```

Author: @simroux thoughts on that change?

Select "Save" once study information has been complete.

### Select Fields

TODO: Add Image

The Field Notes App allows the submitter to select the metadata terms (TODO what word?!) that will be visible and available on the app and when working in the field. 

Available terms can be selected and de-selected individually or in bulk based on the section they're assigned to. The available sections are: "Common", terms commonly measured at the time of sample collection; "Occasional", terms that may sometimes be measured at the time of sample collection; "Uncommon", terms that are rarely measured at the time of sample collection. This allows the submitter to tailor their metadata interface to be more focused for field collection.

If you are completing metadata for multiple sample types in multiple templates you will need to select your fields for each individual template. 
We recommend completing all "New Study" steps prior to and in preparation of field work.

### Samples

In Field Notes, sample metadata are entered individually in a form like structure. This is different than the grid format of the Submission Portal

> The Sample (TODO Biosample in Submission Portal vs Sample here?) Metadata interface consists of a form like structure where metadata is entered individually, unlike the grid format of the Submission Portal.

Reviewer 2: I would maybe only keep this one ("The sample ... Portal") or the previous one ("In Filed Notes ... Portal"), but not both ?

Author: @simroux  I don't follow you herE?

Reviewer 2: I think this was just to say that I don't think we need both sentences, so we should just pick one (but I don't have a strong feeling on which) ?

> Required metadata is marked with a ⚠️ symbol. Though, no metadata can be provided and the same will still be saved.

Reviewer 2: maybe "no metadata can be provided" -> "these fields can be left empty" ?

Author: ```suggestion
Required metadata is marked with a ⚠️ symbol. Though, these fields can be left empty and the sample will still be saved.
```

Author: @simroux like that?

Once a sample has been started, you can return to the Study page via the ← button in the upper left. Your new sample will now appear at the bottom of your screen.

Additional information about sample metadata can be found above in [header](#sample-metadata). 

#### Tools and Features 

##### Sorting Samples

When samples are added to a study, they are listed at the bottom of the submission. Each sample can be revisited or viewed by selecting the desired sample. Additionally, samples can be filtered based on the sample type / template or you can search for a specific sample.

##### Term Information

[![](../_static/images/howto_guides/portal_guide/sub_portal_enviro_package.png)](../_static/images/howto_guides/portal_guide/sub_portal_enviro_package.png)

[![](../_static/images/howto_guides/portal_guide/column_help.png)](../_static/images/howto_guides/portal_guide/column_help.png)

Similar to [header](#column-information) for the Submission Portal, when you select a term to complete metadata, term help will be included below the input field. This provides a description of the term, additional guidance, and examples of valid completion. 

##### Location Provided Metadata

> Field Notes can use your mobile device information and location to populate some metadata fields. For example, selecting "Set to today" for the term "collection date" will auto populate with the current date. Additional, mobile GPS coordinates can be used to populate "latitude and longitude" field by selecting "Use GPS location". This along with other auto-populating fields ensure accurate and structured metadata in real time and minimuzes human error.

Reviewer 2: "Additional" -> "Additionally" ?
"minimuzes" -> "minimizes"

Author: ```suggestion
Field Notes can use your mobile device information and location to populate some metadata fields. For example, selecting "Set to today" for the term "collection date" will auto populate with the current date. Additionally, mobile GPS coordinates can be used to populate "latitude and longitude" field by selecting "Use GPS location". This along with other auto-populating fields ensure accurate and structured metadata in real time and minimizes human error.
```

All fields in the metadata template innately fall into one of three color categories: Grey, Yellow, or Purple. Grey, or no highlighted color, indicates optional fields. Required columns are denoted with yellow. These yellow columns must be correctly completed or the submission will not validate. Recommended fields, highlighted purple, are required where applicable. If any of the purple columns provide information relevant to the study, they should be completed. For example, if samples are from a moisture manipulation study, the column ‘watering regimen’ should be filled in to provide context and information about the samples. 

After selecting the validate button, cells will become color coded to indicate invalid and incomplete metadata. Dark pink cells indicate a required cell has been left empty. Light pink indicates that there is an error in the formatting of the information entered into that cell. The "Column Information" tool, described above, provides expected structure patterns and examples of valid metadata, which you can refer to when addressing validation errors.

## Field Notes Functionality

https://microbiomedata.org/field-notes/

> The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices. It's currently available for beta testing. See (TODO) to sign up!

Author: @eecavanna or @pkalita-lbl 
How do I link another page / file?? I want to just point to field_notes_tester.md in this (TODO)

> The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices. It's currently available for beta testing. See (TODO) to sign up!

Author: ```suggestion
The NMDC Field Notes app is for mobile device use. It runs on both Apple (iOS) and Android devices. It's currently available for beta testing. See [link text](./field_notes_tester) to sign up!
```

The "Guide" section of the app contains recommendations about fieldwork in general, and about collecting metadata for specific sample type(s). We recommend viewing it before going out into the field.

TODO: Add screen shot images

The NMDC Field Notes app requires ORCiD authentication to access it. 

Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 

TODO: Update all these to field notes!

(Step 1 of 3) Welcome to the NMDC Field Notes mobile app! 

[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)

Once signed in with an ORCiD, you will have the ability to access existing studies or create a study using the 'New’ button. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point. 

### New Study

[![](../_static/images/howto_guides/portal_guide/study_info.png)](../_static/images/howto_guides/portal_guide/study_info.png)

> The Study Information page requires a valid ‘Study Name’ and a valid email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [header](#submission-portal-functionality) [header](#study) for more details about Study information.

Author: ```suggestion
The Study Information page requires a valid ‘Study Name’ and a valid email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [header](#submission-portal-functionality) [Study](#study) for more details about Study information.
```

Author: I thought it had to be header? but I made it study and we'll see!

> Also in the New Study you will select the "Environment Template" that best fits your samples. The templates available correspond to the GSC Environmental Extensions. More information is available above in sections [header](#environmental-package). Multiple environments can be selected and submitted in one submission.

Author: ```suggestion
Also in the New Study you will select the "Environment Template" that best fits your samples. The templates available correspond to the GSC Environmental Extensions. More information is available [above](#environmental-extension). Multiple environments can be selected and submitted in one submission.
```

Select "Save" once study information has been complete.

### Select Fields

TODO: Add Image

> The Field Notes App allows the submitter to select the metadata terms (TODO what word?!) that will be visible and available on the app and when working in the field. 

Author: @pkalita-lbl @eecavanna suggestions on the word here?
metadata fields
metadata terms 
?

Reviewer 3: We use the term "fields" in the app itself.

> The Field Notes App allows the submitter to select the metadata terms (TODO what word?!) that will be visible and available on the app and when working in the field. 

Author: ```suggestion
The Field Notes App allows the submitter to select the metadata fields that will be visible and available on the app and when working in the field. 
```

Available terms can be selected and de-selected individually or in bulk based on the section they're assigned to. The available sections are: "Common", terms commonly measured at the time of sample collection; "Occasional", terms that may sometimes be measured at the time of sample collection; "Uncommon", terms that are rarely measured at the time of sample collection. This allows the submitter to tailor their metadata interface to be more focused for field collection.

If you are completing metadata for multiple sample types in multiple templates you will need to select your fields for each individual template. 
We recommend completing all "New Study" steps prior to and in preparation of field work.

### Samples

> In Field Notes, the sample (TODO Biosample in Submission Portal vs Sample here?) metadata interface consists of a form like structure where metadata is entered individually, unlike the grid format of the Submission Portal.

Author: ```suggestion
In Field Notes, the sample metadata interface consists of a form like structure where metadata is entered individually, unlike the grid format of the Submission Portal.
```

> In Field Notes, the sample (TODO Biosample in Submission Portal vs Sample here?) metadata interface consists of a form like structure where metadata is entered individually, unlike the grid format of the Submission Portal.

Author: @eecavanna @pkalita-lbl 

Thoughts on just saying sample here?

Reviewer 3: The app always refers to them as just "samples"

> Required metadata is marked with a ⚠️ symbol. Though, these fields can be left empty and the sample will still be saved.

Reviewer 3: This is tricky because a missing value for a required field is only one of many reasons why the warning icon might get shown. So, maybe something like this.

```suggestion
Fields marked with a ⚠️ symbol indicate a validation issue. For example, a required value is missing or a value is not in the expected format. These do not prevent your updates from being saved. The issues only need to be resolved before the final submission to NMDC for review.
```

Once a sample has been started, you can return to the Study page via the ← button in the upper left. Your new sample will now appear at the bottom of your screen.

> Additional information about sample metadata can be found above in [header](#sample-metadata). 

Reviewer 3: > header

> Additional information about sample metadata can be found above in [header](#sample-metadata). 

Author: ```suggestion
Additional information about sample metadata can be found above in [Sample Metadata](#sample-metadata). 
```

#### Tools and Features 

##### Sorting Samples

When samples are added to a study, they are listed at the bottom of the submission. Each sample can be revisited or viewed by selecting the desired sample. Additionally, samples can be filtered based on the sample type / template or you can search for a specific sample.

##### Term Information

[![](../_static/images/howto_guides/portal_guide/sub_portal_enviro_package.png)](../_static/images/howto_guides/portal_guide/sub_portal_enviro_package.png)

[![](../_static/images/howto_guides/portal_guide/column_help.png)](../_static/images/howto_guides/portal_guide/column_help.png)

> Similar to [header](#column-information) for the Submission Portal, when you select a term to complete metadata, term help will be included below the input field. This provides a description of the term, additional guidance, and examples of valid completion. 

Reviewer 3: > header

> Similar to [header](#column-information) for the Submission Portal, when you select a term to complete metadata, term help will be included below the input field. This provides a description of the term, additional guidance, and examples of valid completion. 

Author: ```suggestion
Similar to [column information](#column-information) for the Submission Portal, when you select a term to complete metadata, term help will be included below the input field. This provides a description of the term, additional guidance, and examples of valid completion. 
```

##### Location Provided Metadata

Field Notes can use your mobile device information and location to populate some metadata fields. For example, selecting "Set to today" for the term "collection date" will auto populate with the current date. Additionally, mobile GPS coordinates can be used to populate "latitude and longitude" field by selecting "Use GPS location". This along with other auto-populating fields ensure accurate and structured metadata in real time and minimizes human error.

##### Real Time Validation  

[![](../_static/images/howto_guides/portal_guide/validate.png)](../_static/images/howto_guides/portal_guide/validate.png)

Real time validation available in Field Notes is similar to that in the Submission Portal [header](#real-time-validation). No color coding, but the ⚠️ will disappear when metadata is valid, and a note below the input box will provide if a required field is missing or if a field is misformatted. 

> ##### OTHER TOOLS IN APP?

Reviewer 3: I don't think so?
